soc/espressif/esp32/soc.c

```diff
@@ -26,6 +27,15 @@ void IRAM_ATTR __esp_platform_app_start(void)
 
 	esp_flash_config();
 
+#if CONFIG_ESP32_EFUSE_VIRTUAL
```

Comment:
Can we please hide and unite the efuse init code in `soc/common/include/esp_efuse.h` using only a single call? That way, we don't bloat the `soc.c` each time we add a new feature.